.github/ISSUE_TEMPLATE/MCPSERVER.yml

```diff
@@ -0,0 +1,91 @@
+name: NuGet MCP Server Issue
+description: Report a bug, suggestion, or behavior discrepancy for the NuGet MCP Server (Model Context Protocol)
+title: "[MCP Server]: "
+labels: ["Product:NuGetMCPServer", "Triage:Untriaged"]
```

Comment:
We have to create `Product:NuGetMCPServer` if we agree on this label.

Reply:
I would vote for just `Product:MCPServer`

Reply:
I agree. The word `NuGet` is redundant. Removed it in the latest commit.

Reply:
I have created the label